metricbeat/module/elasticsearch/cluster_stats/_meta/data.json

```diff
@@ -0,0 +1,55 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "elasticsearch": {
+        "cluster": {
+            "id": "njqU4EQaTROIDlWPeUMQyw",
+            "name": "elasticsearch",
+            "stats": {
+                "indices": {
```

Comment:
How does the cluster index data overlap with the `index` and `node_stats` metricset data?

Reply:
* `index` metricset has `docs`, `segments`, and `store` which are also present in this metricset here.

* `node_stats` metricset has `docs`, `segments`, and `store`, which are also present in this metricset here.

* `node_stats` metricset also has `fs` and `jvm` stats, which are currently not present in this metricset here but are available from the Elasticsearch `GET _cluster/stats` API.

* This metricset here has `fielddata` which is absent in the `index` metricset but is available from the Elasticsearch `GET _stats` API. 

* This metricset here has `fielddata` which is absent in the `node_stats` metricset and also not available from the Elasticsearch `GET _nodes/_local` API. However it is available from the Elasticsearch `GET _nodes/_local/stats` API. Of course, the latter API is missing a bunch of _other_ fields that are present in the former API so either we call both or we ask the ES team to enhance the latter API's response to include all the fields we use from the former API's response as well.

Reply:
I do need to fix the field names in this metricset here so they match up with the ones in other `elasticsearch` metricsets (e.g. I need to use `count` instead of `total`, etc.)

Reply:
Field names fixed in 6662787.

Reply:
Should we report all the data we have in the index and node_stats metricset here too? Or should we focus on the data points that other metricsets don't have?

Reply:
I think to start off we can only report the stats that are not already covered by other metricsets. Concretely, those would be `fielddata` and `shards` fields.

Reply:
++

Reply:
Fixed in 27ef3fb.